pkg/bgp/controller/subscriber.go

```diff
@@ -0,0 +1,217 @@
+// Copyright 2021 Authors of Cilium
```

Comment:
Should we add MetalLB authors to this module authors? I guess lots of code is taken from MetalLB?

Reply:
Done